### PR TITLE
Add codecov badge

### PR DESCRIPTION
**Pull request by JoshuaSBrown**

### CODE REVIEW

**Review by junghans — CHANGES_REQUESTED** [review]
Huh, https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists aren't list just:
```
*
*
*
```
without a newline?

**Comment by junghans** [comment]
Do you want to add travis and gitlab badges, while you are at it?

**Comment by JoshuaSBrown** [comment]
@junghans Maybe in a separate pull request. This one has been annoying. The auto correct features for .md files are not exhaustive in their formatting corrections.

**Comment by JoshuaSBrown** [comment]
Markdown has this weird rule that all list items must be tight unless any one item extends beyond 80 characters and wraps onto the next line than all items in the list must be loose. Super annoying... and kind of ugly.

**Comment by junghans** [comment]
Can you shorten the long ones?

**Comment by JoshuaSBrown** [comment]
I'm actually wondering if there is a way to ignore that particular rule.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,246 +1,261 @@
-For more detailed information about the changes see the history of the [repository](https://github.com/votca/csg/commits/master).
+For more detailed information about the changes see the history of the
+[repository](https://github.com/votca/csg/commits/master).
 
 ## Version 1.6-dev
 
-## Version 1.5 _SuperVictor_ (released 31.01.19)
- * enable gitlab CI
+## Version 1.5 *SuperVictor* (released 31.01.19)
```
Comment (junghans) [inline CHANGELOG.md]: Huh, no italic in headers?

Comment (JoshuaSBrown) [reply CHANGELOG.md]: I'm still trying to figure out how to use the different auto formatters. It appears that no one autoformatter can do everything.

**Review by junghans — APPROVED** [review]

**Pull request merged**